### ISSUE TITLE
recent change broke python versioning

### ISSUE THREAD

**Issue by joshStillerman** [MDSplus/mdsplus#1697]
The changes to make the git version differently have caused the python version in the kit to be unlabeled:
```
jas@MFEWS02:~$ python
Python 2.7.15rc1 (default, Nov 12 2018, 14:31:15) 
[GCC 7.3.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> from MDSplus import Tree
PYTHONPATH was set to: /usr/local/mdsplus/pydevices and unable to import version information
Warning:
  The MDSplus python module version (Unknown) does not match
  the version of the installed MDSplus libraries (7.76.1).
  Upgrade the module using the mdsplus/mdsobjects/python directory of the
  MDSplus installation.
>>> 
```

**Comment by AndreaRigoni** [comment]
Ups .. ok I missed to change the __init__.py check.  Fixing now

**Comment by AndreaRigoni** [comment]
fixed in #1701
